articles/extensions/delegated-admin/hooks.md

```diff
@@ -299,6 +314,145 @@ The **Settings Query** allows you to customize the look and feel of the extensio
+    // The dictionary allows you to overwrite the title of the dashboard and the "Memberships" label in the Create User dialog.
+    dict: {
+      title: department ? department + ' User Management' : 'User Management Dashboard',
+      memberships: 'Departments',
+      menuName: ctx.request.user.name
+    },
+    // The CSS option allows you to inject a custom CSS file depending on the context of the current user (eg: a different CSS for every customer)
+    css: (department && department !== 'IT') && 'https://rawgit.com/auth0-extensions/auth0-delegated-administration-extension/master/docs/theme/fabrikam.css'
+  });
+}
+```
+
+## Custom Fields
+
+Starting with version 3.0 of the Delegated Admin Extension you can now specify custom fields.  These custom fields can be stored anywhere in user_metadata or app_metadata when creating,
+updating, or displaying users.  You can use these fields to search for users and display them in the search results.  Also, it allows you to customize existing fields, such as email, username,
+name, and connection.
+
+To utilize custom fields, you must add a list of userFields to the **Settings Query**.
+
+Schema for a user field:
+```js
+userFields: [
+    {
+        "property": string, // required
+        "label": string,
+        "sortProperty": string,
+        "display": true || function.toString(),
+        "search": false || {
+            "display": true || function.toString()
+            "listOrder": 1,
+            "listSize": string(###%), // e.g. 15%
+            "filter": boolean,
+            "sort": boolean
+        },
+        "edit": false || {
+            "display": true || function.toString()
+            "type": "text || select || password",
+            "component": "InputText || Input Combo || InputMultiCombo || InputSelectCombo",
+            "options": Array(string) || Array ({ "value": string, "label": string }),
+            "disabled": true,
+        },
+        "create": false || {
+            "display": true || function.toString()
+            "type": "text || select || password",
+            "component": "InputText || Input Combo || InputMultiCombo || InputSelectCombo",
+            "options": Array(string) || Array ({ "value": string, "label": string }),
+            "disabled": true,
+        }
+    },
+    ...
+]
+```
+
+- **property** (**required**): This is the property name of the ctx.payload object for the write hook.
+    - Example: `"property": "app_metadata.dbId"` will get set in `ctx.payload.app_metadata.dbId` in the write hook
+- **label**: This is the label that will be used when adding a label for the field on the user info page, create page, edit profile page, or search page
+- **sortProperty**: If sorting by a different field than this for the search table, use this field.  Dot notation is allowed.
+- **display**: true || false || stringified => This is the default display value.  If not overridden in search, edit, or create, it will use this value.
```

Comment:
It would be good to define the default values. I'm assuming is `true` but I'm not a 100% sure from reading this

Reply:
Fixed